packages/palette-charts/package.json

```diff
@@ -13,7 +13,6 @@
     "clean": "rm -rf dist",
     "compile": "babel src --source-maps --extensions '.ts,.tsx' --ignore src/**/__tests__,src/**/__stories__ --out-dir dist",
     "lint": "tslint --project tsconfig.json 'src/*/**'",
-    "prepublishOnly": "yarn clean && yarn compile && yarn type-declarations",
```

Comment:
When I was upgrading things to React 18 I had to hand publish this one -- couldn't figure out what was going on

Reply:
Yeah, Not sure how to debug this. But I've removed that commit since I don't need the canary anymore 🤷🏽